experiment/resultstore/main.go

```diff
@@ -102,7 +102,7 @@ func parseOptions() options {
 }
 
 func main() {
-	logrusutil.ComponentInit("storeship")
```

Comment:
arguably this one is correct though :) what is storeship?

Reply:
Must have forgotten to make the name incomprehensible
https://www.merriam-webster.com/dictionary/storeship